src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -6270,69 +6270,49 @@ private static string GetNamespaceToRemove(CompletionContext context, TypeComple
         internal static List<CompletionResult> CompleteHelpTopics(CompletionContext context)
         {
             var results = new List<CompletionResult>();
-            var searchPaths = new List<string>();
-            var currentCulture = CultureInfo.CurrentCulture.Name;
-
-            // Add the user scope path first, since it is searched in order.
-            var userHelpRoot = Path.Combine(HelpUtils.GetUserHomeHelpSearchPath(), currentCulture);
-
-            if (Directory.Exists(userHelpRoot))
-            {
-                searchPaths.Add(userHelpRoot);
-            }
-
-            var dirPath = Path.Combine(Utils.GetApplicationBase(Utils.DefaultPowerShellShellID), currentCulture);
-            searchPaths.Add(dirPath);
-
-            var wordToComplete = context.WordToComplete + "*";
-            var topicPattern = WildcardPattern.Get("about_*.help.txt", WildcardOptions.IgnoreCase);
-            List<string> files = new List<string>();
+            string userHelpDir = HelpUtils.GetUserHomeHelpSearchPath();
+            string appHelpDir = Utils.GetApplicationBase(Utils.DefaultPowerShellShellID);
+            string currentCulture = CultureInfo.CurrentCulture.Name;
+
+            //search for help files for the current culture + en-US as fallback
+            var searchPaths = new string[4]
+            { 
+                Path.Combine(userHelpDir, currentCulture),
+                Path.Combine(userHelpDir, "en-US"),
+                Path.Combine(appHelpDir, currentCulture),
+                Path.Combine(appHelpDir, "en-US")
+            }.Distinct();
+
+            string wordToComplete = context.WordToComplete + "*";
+            var foundTopics = new SortedSet<string>(StringComparer.OrdinalIgnoreCase);
 
             try
             {
                 var wildcardPattern = WildcardPattern.Get(wordToComplete, WildcardOptions.IgnoreCase);
 
                 foreach (var dir in searchPaths)
                 {
-                    foreach (var file in Directory.EnumerateFiles(dir))
+                    var currentDir = new DirectoryInfo(dir);
+                    if (currentDir.Exists)
                     {
-                        if (wildcardPattern.IsMatch(Path.GetFileName(file)))
+                        foreach (var file in currentDir.EnumerateFiles("about_*.help.txt"))
                         {
-                            files.Add(file);
+                            if (wildcardPattern.IsMatch(file.Name))
+                            {
+                                string topicName = file.Name.Substring(0, file.Name.LastIndexOf(".help.txt"));
+                                bool notDuplicate = foundTopics.Add(topicName);
+                                if (notDuplicate)
+                                {
+                                    results.Add(new CompletionResult(topicName));
+                                }
```

Comment:
Removing duplications changes current behavior. For the PR this looks as unrelated change. It seems we have an issue for the scenario and we shouldn't remove duplicates but allow users to select. I suggest revert the change in the PR.

Reply:
Duplicates serve no purpose, if we don't filter them out here then PSReadline will do it for us and if that doesn't do it then the user will simply end up with 2 identical tab completion results and probably assume something is broken.  
Get-Help would have to be updated to support specifying a different language and I think that's a good idea, but I think it should be done with another parameter, not by cluttering up tab completion suggestions with near identical suggestions.

Reply:
See #11205. I suggest to address the issue in separate PR and keep current behavior in the PR.